gui/chiaki.desktop

```diff
@@ -2,6 +2,6 @@
 Type=Application
 Name=Chiaki
 Comment=PlayStation 4 Remote Play Client
-Exec=chiaki
+Exec=env QT_QPA_PLATFORM=xcb chiaki
```

Comment:
This is not needed anymore as the EGL (Wayland) support is now fixed.

Reply:
Great news! I revert this workaround then.